package.json

```diff
@@ -74,8 +74,8 @@
     "exportVarName": "Marionette.Toolkit"
   },
   "peerDependencies": {
-    "backbone.marionette": "^2.4.5",
-    "backbone": "^1.3.2",
+    "backbone.marionette": "3.0.0-pre.5",
```

Comment:
This can just be set to "^3.0.0"
I think we can be safe with setting the peer Deps on backbone to `^1.3.0` as well.